### DISCUSSION TITLE
D0xing score

### DISCUSSION THREAD

**Discussion by toniengelhardt** [toniengelhardt/merklin#25] [Ideas]
Would be nice to somehow have an indicator of whether or not a wallet was d0xed in the past and how.

This is just a brainstorm of how we could do it and what the relevant metrics are, e.g. 

- Incoming transactions from KYCd exchange
- Incoming transactions from d0xed wallets
- Connection to Twitter/Discord
- Connection to phone/email
- etc.

**Comment by RyanPaulGannon** [comment]
Hadn't heard of doxing before! Quite interesting after seeing some stuff on Twitter with Elon Musk.

Haven't much to add here, however, in terms of connecting to Twitter, email etc, do you think using Supabase here would work?

**Reply by toniengelhardt** [reply]
I think pseudonymity is very important for web3, but unfortunately many projects encourage users to link Twitter accounts etc. to their wallets. 

I haven't used Supabase yet but I think, if possible, it's best to avoid any centralized backend as long as possible. I'm not sure how [guild.xyz](https://guild.xyz) etc. is doing it, but probably via Twitter API.

D0xing score is not super high priority atm for me, but I think it would be a cool metric somehow and I haven't seen it anywhere else so far.

**Reply by RyanPaulGannon** [reply]
Totally overlooked the centralisation aspect of Supabase! I agree, wherever possible would be best to use something decentralised.

**Reply by toniengelhardt** [reply]
Yes, especially to avoid paying servers...
